browser/html/cool.html.m4

```diff
@@ -180,35 +180,37 @@ m4_ifelse(MOBILEAPP,[true],
     </div>
 
     <!-- Remove if you don't want the About dialog -->
-    <div id="about-dialog" tabIndex="0">
-      <div id="about-dialog-header">
-        <fig id="integrator-logo"></fig>
-        <h1 id="product-name">Collabora Online</h1>
-      </div>
-      <hr/>
-      <div id="about-dialog-container">
-        <div id="about-dialog-logos">
-          <fig id="product-logo"></fig>
-          <fig id="lokit-logo"></fig>
+    <dialog>
```

Comment:
looks ok, but don't you need to use "open" function with it in the JS? (to actually use the feature fully)

Reply:
About dialog code copies the content of the dialog (not the dialog element). We don't need to implement that for now.